static/js/main.js

```diff
@@ -240,11 +237,39 @@ function addScreenshot(screenshot) {
                             data-screenshot='${screenshot.id}'
                             value='${time_input_value}' />
                 </div>
+                <button onclick='flipImage(this, "${screenshot.id}")'>Flip Image</button>
             </div>
         </div>
     `;
-
+    
     document.querySelector('.screenshot-area').insertAdjacentHTML('beforeend', html);
+    
+    // Find canvas, create image
+    let canvas_list = document.querySelectorAll('.image-area');
+    let canvas = canvas_list[canvas_list.length - 1]
+    let context = canvas.getContext('2d');
+    let img = new Image();
+    img.src = screenshot.dataURL;
+    // Add the image to the newly added canvas
+    img.onload = function() {
+        context.drawImage(img, 0, 0);
+    };
```

Comment:
This comment goes along with the comment about using a `canvas` instead of an `img`.


Here I see several lines of work (including waiting for an image to load) instead of the simple process of directly assigning the images source.
We always want to try for KISS ("Keep It Super Simple").